packages/next/cli/next-lint.ts

```diff
@@ -120,7 +122,7 @@ const nextLint: cliCommand = (argv) => {
           --cache-location path::String  Path to the cache file or directory - default: .eslintcache
         
         Miscellaneous:
-          --no-error-on-unmatched-pattern  Prevent errors when pattern is unmatched - default: false
+          --error-on-unmatched-pattern   Show errors when any file patterns are unmatched - default: false
```

Comment:
Technically a breaking API change, but the previous behavior was faulty (empty file directories or folders with all files ignored by ESLint would fail `next lint`) so I don't think we have to wait for the next major version to include this fix.